package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-plugin-typescript-sort-keys",
-  "version": "0.3.0",
+  "version": "0.4.0",
```

Comment:
I only bumped this because it was out of sync with npm (current published version is `0.4.0`)

<details>
    <summary>
        <code>$ npm info eslint-plugin-typescript-sort-keys</code>
    </summary>

    eslint-plugin-typescript-sort-keys@0.4.0 | ISC | deps: 2 | versions: 6
    Sort interface and string enum keys
    https://github.com/infctr/eslint-plugin-typescript-sort-keys#readme
    
    keywords: eslint, eslintplugin, eslint-plugin
    
    dist
    .tarball: https://registry.npmjs.org/eslint-plugin-typescript-sort-keys/-/eslint-plugin-typescript-sort-keys-0.4.0.tgz
    .shasum: 71b5c570fbed41a685ec8905219f4cfdbe2c6dff
    .integrity: sha512-kmIbs5CYYUtYUpZbFyaBWlI+10WDZxfeVlVTgtKkGoPChHBz/Sm+/deeJpO93peI+79dkseFKsZKsJEfbfh+2g==
    .unpackedSize: 12.3 kB
    
    dependencies:
    natural-compare-lite: ~1.4.0 requireindex: ~1.2.0
    
    maintainers:
    - infctr <infctr@gmail.com>
    
    dist-tags:
    latest: 0.4.0
    
    published 3 months ago by infctr <infctr@gmail.com>
</details>